src/index.ts

```diff
@@ -0,0 +1,78 @@
+const inputElement = document.getElementById("commentInput");
+const formElement = document.getElementById("inputForm");
+const ulElement = document.getElementById("commentsList");
+
+//submit form
+if (formElement !== null) {
+    formElement?.addEventListener("submit", (event) => {
+        event.preventDefault();
+        if (inputElement !== null) {
+            const newMsg = (inputElement as HTMLInputElement).value;
+            if (newMsg === "") {
+                return;
+            }
+            // li father
```

Comment:
Our code should be structured in such a way that these comments will be unesscary

If this script was composed of smaller method it would be possible to infer whats happening just by looking at the called methods

Reply:
dont know what you ment, but I did change thing to prevent code duplication so it might solve it